src/main/java/frc/robot/structures/Superstructure.java

```diff
@@ -62,4 +65,8 @@ public boolean isAtPose(Pose2d targetBluePose) {
 		return isAtXAxisPosition(targetBluePose.getX()) && isAtYAxisPosition(targetBluePose.getY()) && isAtAngle(targetBluePose.getRotation());
 	}
 
+	public Command setState(RobotState robotState) {
+		return new WaitCommand(3);
```

Comment:
r u sure?

Reply:
It's a placeholder as there are no states in the template